src/multivariate/mvlogitnormal.jl

```diff
@@ -0,0 +1,146 @@
+"""
+    MvLogitNormal
+
+The [multivariate logit-normal distribution](https://en.wikipedia.org/wiki/Logit-normal_distribution#Multivariate_generalization)
+is the multivariate generalization of [`LogitNormal`](@ref).
+
+It is a more general alternative to [`Dirichlet`](@ref) for probability vectors.
```

Comment:
Why this removal? This is primarily explaining its significance and utility. The compositional data analysis literature describes 3 different notions of independence that exist in distributions on the simplex. Dirichlet exhibits the most extreme versions of all 3 and is provably the only distribution that does so. In that sense, Dirichlet is very weird. It can't even have non-convex equi-probability contours.

While MvLogitNormal does not contain Dirichlet as a special case, it comes close but also for a dense covariance matrix exhibits none of the notions of dependence, and it can have concave contours.

Reply:
It's mostly because I think the other change I recommended conveys the same information--"is a multivariate generalization of [`LogitNormal`](@ref) capable of handling correlations between variables"--but without the possibility of confusing users into thinking this is related to the [generalized Dirichlet distribution](https://www.sciencedirect.com/science/article/pii/S0096300397101400), or into thinking this shares some of the key properties of the Dirichlet distribution (like neutrality).